### PR TITLE
add hostname in dashboard title

### PR DESCRIPTION
**Pull request by steefmin**
### What

A simple JS line that adds the hostname in the title. 

### Why

If you, just as me, have multiple CIMonitors open in one browser, it is almost impossible to distinguish between them. This PR gives you more information.

### CODE REVIEW

File: front-end/index.html [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
         <script type="text/javascript">
             window.CIMonitorVersion = '---version---';
+            document.title = document.title + ' | '+ location.host;
```
Comment (RickvdStaaij) [inline front-end/index.html]: ES6 can concatinate strins like:
```js
document.title = `CIMonitor | ${location.host}`;
```

Comment (RickvdStaaij) [reply front-end/index.html]: This is now placed outside of the `dashboard.js`, I would rather see it inside of `front-end/dashboard.js::created()`. That way it's more part of the application.

Comment (steefmin) [reply front-end/index.html]: moved it

**Comment by RickvdStaaij** [comment]
Codewise looking good! Checking functionally

**Pull request merged**